### PR TITLE
Added questions and results page

### PR DESCRIPTION
**Pull request by venkateshwarans**
### Results - https://deploy-preview-2--buzzzr.netlify.app/pages/results.html

This PR adds MCQ questions and Results page

### CODE REVIEW

**Review by vrinda-mahajan — APPROVED** [review]
The code is clean. Good to merge.

File: css/global.css [hunk 1]
```diff
@@ -14,56 +15,131 @@
...
+label.btn.btn.btn-danger.checked {
+  background-color: var(--red)
+}
+
 /* button */
 
-.btn {
+.btn,
+input[type=checkbox].btn {
   -webkit-box-shadow: 0 9px 0 0 rgb(0 0 0 / 6%);
```
Comment (MayankKumar10) [inline css/global.css:46]: make a helper class of box-shadow so you don't have to repeat it

**Review by MayankKumar10 — APPROVED** [review]
Everything else Looks Good  to me

File: css/global.css (hunk 1 above)
Comment (venkateshwarans) [reply css/global.css:46]: Yeah. But it is of a different color shadow.

Comment (venkateshwarans) [reply css/global.css:46]: Thanks for actually bringing it up. After some bit of googling, I figured out a way to achieve it. https://blog.jim-nielsen.com/2021/css-relative-colors/

**Pull request merged**